TemplateApplication/Account/AccountSheet.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import SpeziAccount
+import SpeziLicense
```

Comment:
We need to pass in the license of the current project as well as the project name to the package list, otherwise each app will display the MIT License and the Spezi Template Application name.